slmigrate/arghandler.py

```diff
@@ -3,50 +3,43 @@
 
 
 # Setup available command line arguments
-def parse_arguments(args):
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--" + constants.capture_arg, help="capture is used to pull data and settings off SystemLink server", action="store_true", )
-    parser.add_argument("--" + constants.restore_arg, help="restore is used to push data and settings to a clean SystemLink server. ", action="store_true", )
-    parser.add_argument("--" + constants.tag.arg, "--tags", "--tagingestion", "--taghistory", help="Migrate tags and tag histories", action="store_true", )
-    parser.add_argument("--" + constants.opc.arg, "--opcua", "--opcuaclient", help="Migrate OPCUA sessions and certificates", action="store_true")
-    parser.add_argument("--" + constants.fis.arg, "--file", "--files", help="Migrate ingested files", action="store_true")
-    parser.add_argument("--" + constants.testmonitor.arg, "--test", "--tests", "--testmonitor", help="Migrate Test Monitor data", action="store_true")
-    parser.add_argument("--" + constants.asset.arg, "--assets", help="Migrate asset utilitization and calibration data", action="store_true")
-    parser.add_argument("--" + constants.repository.arg, "--repo", help="Migrate packages and feeds", action="store_true")
-    parser.add_argument("--" + constants.alarmrule.arg, "--alarms", "--alarm", help="Migrate Tag alarm rules", action="store_true")
-    parser.add_argument("--" + constants.userdata.arg, "--ud", help="Migrate user data", action="store_true")
-    parser.add_argument("--" + constants.notification.arg, "--notifications", help="Migrate notifications strategies, templates, and groups", action="store_true")
-    parser.add_argument("--" + constants.states.arg, "--state", help="Migrate system states", action="store_true")
-    parser.add_argument("--" + constants.migration_arg, "--directory", "--folder", help="Specify the directory used for migrated data", action="store", default=constants.migration_dir)
-    parser.add_argument("--" + constants.thdbbug.arg, help="Migrate tag history data to the correct MongoDB to resolve an issue introduced in SystemLink 2020R2 when using a remote Mongo instance. Use --sourcedb to specify a source database. admin is used if none is specfied", action="store_true")
-    parser.add_argument("--" + constants.source_db_arg, "--sourcedb", help="The name of the source directory when performing intra-databse migration", action="store", default=constants.source_db)
-    return parser
-
+def parse_arguments():
```

Comment:
Why was `args` removed? This tool doesn't actually run with this removed. 
```
PS C:\Users\markblack\Documents\GitHub\systemlink-migration-sandbox> python .\systemlinkmigrate.py restore --test
Traceback (most recent call last):
  File ".\systemlinkmigrate.py", line 9, in <module>
    arguments = arghandler.parse_arguments(sys.argv[1:]).parse_args()
TypeError: parse_arguments() takes 0 positional arguments but 1 was given
```



Keep in mind that the test as structured now are unit tests not integration tests. Those need to be run by hand - basically just calling the tool like an end user.